src/Files.App/Helpers/Dialog/DynamicDialogFactory.cs

```diff
@@ -266,6 +266,53 @@ public static DynamicDialog GetFor_GitCheckoutConflicts(string checkoutBranchNam
 			return dialog;
 		}
 
+		public static DynamicDialog GetFor_GitMergeConflicts(string checkoutBranchName, string headBranchName)
+		{
+			DynamicDialog dialog = null!;
+
+			var optionsListView = new ListView()
+			{
+				ItemsSource = new string[]
+				{
+					string.Format(Strings.AbortMergeAndSwitch.GetLocalizedResource(), checkoutBranchName),
+					string.Format(Strings.StayAndResolveConflicts.GetLocalizedResource(), headBranchName)
+				},
+				SelectionMode = ListViewSelectionMode.Single
+			};
+			optionsListView.SelectedIndex = 0;
+
+			optionsListView.SelectionChanged += (listView, args) =>
+			{
+				dialog.ViewModel.AdditionalData = optionsListView.SelectedIndex == 0
+					? GitCheckoutOptions.AbortMerge
+					: GitCheckoutOptions.None;
+			};
+
+			dialog = new DynamicDialog(new DynamicDialogViewModel()
+			{
+				TitleText = Strings.SwitchBranch.GetLocalizedResource(),
+				PrimaryButtonText = Strings.OK.GetLocalizedResource(),
```

Comment:
I'm wondering if OK is the correct string for this type of dialog. It's usually used a close button, but in this scenario it's being used to confirm a selection.

Reply:
Dynamically changing the text is not recommended. Unless someone has a better idea, let's go with "Confirm".